blogs/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "blogs",
+  "version": "1.0.0",
+  "main": "converter.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node converter.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "marked": "^4.1.0",
```

Comment:
Do you want to use marked or showed? Remove the one you don't want to use from here, then do another 
```
npm i
```